### ISSUE TITLE
setCustomizedUcrop(UCrop)

### ISSUE THREAD

**Issue by meddd** [Tofira/ImagePickerWithCrop#3]
Hi,
Could you show some example how to use setCustomizedUcrop(UCrop).

**Comment by chaibiskut** [comment]
Yeah I have the same question, I am trying to create new instance of UCrop but its access level is private, so how can i set customized aspect ratio?

**Comment by nikiiTrivedi** [comment]
UCrop.of(sourceUri, destinationUri)
    .withAspectRatio(16, 9)
    .withMaxResultSize(maxWidth, maxHeight)
    .start(context);
